packages/geolocation/src/useCases/place.js

```diff
@@ -1,22 +1,54 @@
 'use strict';
 const boom = require('@hapi/boom');
+const client = require('../drivers/mongodb/connection');
 
 class Place {
-  constructor() {
-    this.places = [
-      { id: 'sdf7s6f4s', lat: 45.46546, lon: -58.15646 },
-      { id: 'asd45qdqs', lat: 86.46546, lon: -34.15646 },
-      { id: 'fs45qqads', lat: 58.46546, lon: -12.15646 },
-      { id: '6a5sd46as', lat: 17.46546, lon: -91.15646 },
-    ];
-  }
+  constructor() {}
```

Comment:
@Edilberto-Vazquez in most of the packages inside geolocation we are not using classes so in order to keep the consistency inside the package we should try to continue using the same paradigm either it be using classes or structures.

Reply:
I changed the class to a closure, can you check if it's correct the way i do?